src/blockdata/block.rs

```diff
@@ -131,6 +141,7 @@ impl XField {
         match self {
             XField::None => 0u8,
             XField::AggregatePublicKey(_) => 1u8,
+            XField::MaxBlockSize(_) => 2u8,
```

Comment:
MaxBlockSize xfield value is 4 bytes, so why is it 9?

Reply:
Sorry! I must have confused that its 8 byte integer. I'll check the tests and fix it